validate_key.py

```diff
@@ -1,37 +1,40 @@
 from datetime import datetime, timedelta
 import hashlib
-
 from pywell.entry_points import run_from_cli, run_from_api_gateway
-
+from pywell.secrets_manager import get_secret
 
 DESCRIPTION = 'Validate a download key.'
 
 ARG_DEFINITIONS = {
-    'KEY': 'Key to validate.',
-    'MAX_AGE': 'Number of days a key should be considered valid.',
-    'SECRET': 'Secret to use for validation.'
+    'KEY': 'Key to validate.'
 }
 
-REQUIRED_ARGS = ['KEY', 'MAX_AGE', 'SECRET']
+REQUIRED_ARGS = ['KEY']
 
 
-def main(args):
+def main(args, script_settings={}):
+    if not script_settings:
+        script_settings = get_secret('ak-partner-rsvp')
+    key_hash_secret = script_settings['KEY_HASH_SECRET']
```

Comment:
Not sure this is really needed since it's only used once

Reply:
sorry, do you mean the hash secret or something else?

Reply:
Yes, the hash secret.